### ISSUE TITLE
Update NodeSchool LinkedIn page

### ISSUE THREAD

**Issue by WaleedAshraf** [nodeschool/discussions#2314]
Hi,

I just visited [NodeSchool LinkedIn](https://www.linkedin.com/company/nodeschool/) page and I think it needs to be updated.
- Add Logo
- Add Intro 
> NodeSchool is an open source project run by volunteers with two goals: to create high quality programming curriculum and to host community learning events.
- Add all the content in [About page](https://nodeschool.io/about.html) on LinkedIn.

And some links like [Ask Quesstion](https://github.com/nodeschool/discussions/issues/new) etc.

**Comment by WaleedAshraf** [comment]
@riyadhalnur Not sure, but maybe you can help with this?

**Comment by usmantahirr** [comment]
If we get domain email we can create one ourselves aswell.

**Comment by WaleedAshraf** [comment]
Ping: @nodeschool/core

**Comment by SomeoneWeird** [comment]
I don't seem to have access to that - not sure who does.

**Comment by RichardLitt** [comment]
@rickheere @icalik You're listed as members on the page; do you have access?

**Comment by rickheere** [comment]
I have no access to the page.

**Comment by D3Portillo** [comment]
I agree with @a0viedo , it's a community, and at linkedin will be weird to be there as a company or somethin, we are from extremely diff places, and in the case it exists will be with a selected bunch of people

**Comment by WaleedAshraf** [comment]
Not sure, but I think there is an option in the LinkedIn page to show the profile as an "Independent Organization" or something else which is the right title for NodeSchool.

If not, the current NodeSchool profile should be closed/deleted.
@a0viedo @D3Portillo

**Comment by riyadhalnur** [comment]
Makes sense if that's possible to do on LinkedIn

**Comment by WaleedAshraf** [comment]
Looking into more details of that page, came to know that it was auto-created by LinkedIn and owned by no one.

I have taken the ownership of the page and added all the details.
Take a look: https://www.linkedin.com/company/nodeschool/

Open for suggestions + more admins. 

As now we have ownership of the page, we can also deactivate it if needed.

**Comment by gokaygurcan** [comment]
I'd ping @nodeschool/core about this.

**Comment by RichardLitt** [comment]
Does @stale actually help in this repository?

In any event, still an issue.
